docs/openapi.json

```diff
@@ -33,11 +33,13 @@
             "description": "Configuration information to consume by Reana-UI.",
             "examples": {
               "application/json": {
+                "admin_email": "team@reanahub.io",
```

Comment:
This email will be fully customizable by any institute having its REANA cluster, thus, what about using a the generic domain `example.org`? Maybe something like `admin@example.org`?